scripts/vagrant-mod-env.sh

```diff
@@ -6,9 +6,12 @@ USER=vagrant
 HOME=/home/${USER}
 
 echo 'Modifying environment'
-chown -R ${USER}:${USER} ${HOME}
+chown -R ${USER}:${USER} ${HOME}/go/src/github.com/kinvolk/kube-spawn/k8s
 chmod +x ${HOME}/build.sh
-setenforce 0
+
+# setenforce always returns 1 when selinux is disabled.
+# we should ignore the error value to continue to rest commands.
```

Comment:
Nit: "... to continue with the remaining commands" or, simpler, just "Ignore the error and continue".